packages/asset-swapper/src/utils/market_operation_utils/aave_reserves_cache.ts

```diff
@@ -0,0 +1,115 @@
+import { logUtils } from '@0x/utils';
+import { gql, request } from 'graphql-request';
+
+import { constants } from '../../constants';
+
+const RESERVES_GQL_QUERY = gql`
+    {
+        reserves(where: { isActive: true, isFrozen: false }, orderBy: totalLiquidity, orderDirection: desc) {
```

Comment:
**TODO** specify number of pools to fetch?

Reply:
IIRC 100 is the default limit, which is more than enough but it's probably better to be explicit :)